api/handlers/node.go

```diff
@@ -78,6 +81,14 @@ type healthCheckJSON struct {
 	} `json:"advanced"`
 }
 
+type SignRequestJSON struct {
+	Data string `json:"data"`
+}
+
+type SignResponseJSON struct {
+	Signature string `json:"signature"`
+}
+
```

Comment:
For consistency, let's put these as anonymous structs in the request handler method, like in:
https://github.com/bloxapp/ssv/blob/c567481d43c0727a81abbaea4846b8a61f7a1df7/api/handlers/validators.go#L22-L29

There's no reason to expose them outside at the moment. When they are used in tests, we can construct them from there as anonymous structs as well (its really only 1 field).

In the future (SSV API v2) we'll have an API `client` package so we'd be able to just call `Sign()` method from tests, rather than encoding JSONs :)